### PR TITLE
Added configuration for conducting mutation testing.

### PR DESCRIPTION
**Pull request by LasTshaMAN**
Decided to play with [mutation testing](http://pitest.org/).

Also, cleaned up pom.xml a bit.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -45,52 +53,17 @@
...
+                        <param>hashCode</param>
+                        <param>equals</param>
+                    </excludedMethods>
                 </configuration>
             </plugin>
-
-            <plugin>
-                <!-- Executing JAR-->
-                <groupId>org.codehaus.mojo</groupId>
-                <artifactId>exec-maven-plugin</artifactId>
```
Comment (ghost) [inline pom.xml:68]: Why did you remove this plugin?

File: pom.xml [hunk 2]
```diff
@@ -45,52 +53,17 @@
                     </execution>
                 </executions>
             </plugin>
-
             <plugin>
-                <!-- Building an executable JAR -->
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-jar-plugin</artifactId>
```
Comment (ghost) [inline pom.xml:52]: Why did you remove this plugin?

**Review by ghost — APPROVED** [review]

File: pom.xml (hunk 2 above)
Comment (LasTshaMAN) [reply pom.xml:52]: We don't need to create executable jar or any jar for that matter as of now.
It was necessary to run a chat server, but now we don't have it.
And SpringBoot will take care of running our app.

File: pom.xml (hunk 1 above)
Comment (LasTshaMAN) [reply pom.xml:68]: We don't need to create executable jar or any jar for that matter as of now.
It was necessary to run a chat server, but now we don't have it.
And SpringBoot will take care of running our app.
